content/templates/profile.md

```diff
@@ -1,26 +1,43 @@
 ---
-created: 2023-11-17
-title: 
+created: "{{DATE}}"
```

Comment:
no quotes i think ...

Reply:
automatic date filling in works as it is so leaving as is